jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -70,6 +71,20 @@ public synchronized static boolean isCRIUSupportEnabled() {
 		return criuSupportEnabled;
 	}
 
+	/**
+	 * Queries if CRIU Security provider needs to be disabled
+	 * when CRIU support is enabled.
+	 *
+	 * @return true if needed, false otherwise
+	 */
+	public synchronized static boolean disableCRIUSecProvider() {
```

Comment:
I don't think `synchronized` is needed here.

Reply:
Updated